importers.py

```diff
@@ -1,10 +1,37 @@
 import csv
 import shapefile
+import os
+from datetime import datetime
 
 from loopstats import LoopStats
+from notifications_python_client.notifications import NotificationsAPIClient
 
 
-class CSVImportCommand(object):
+class ImportCommand(object):
+    def notify_import_completed(self, imported_files):
+        EMAIL_TO_NOTIFY = os.environ.get('EMAIL_TO_NOTIFY')
+        NOTIFY_API_TOKEN = os.environ.get('NOTIFY_API_TOKEN')
+        NOTIFY_EMAIL_TEMPLATE = os.environ.get('NOTIFY_EMAIL_TEMPLATE')
+
+        if EMAIL_TO_NOTIFY and NOTIFY_API_TOKEN and NOTIFY_EMAIL_TEMPLATE:
+            notifications_client = NotificationsAPIClient(NOTIFY_API_TOKEN)
+
+            personalisation = {
+                'import_name': self.__class__.__name__,
+                'completion_time': datetime.now().isoformat(),
+                'imported_files': imported_files,
+                'import_url': self.api_url
+            }
+
+            response = notify_client.send_email_notification(
```

Comment:
ouch! Nice spot! I refactored the code and forgot that line. I'm going to fix it now